apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/service/ReleaseHistoryService.java

```diff
@@ -202,10 +204,11 @@ private void cleanReleaseHistory(ReleaseHistory cleanRelease) {
     boolean hasMore = true;
     while (hasMore && !Thread.currentThread().isInterrupted()) {
       List<ReleaseHistory> cleanReleaseHistoryList = releaseHistoryRepository.findFirst100ByAppIdAndClusterNameAndNamespaceNameAndBranchNameAndIdLessThanEqualOrderByIdAsc(
-          appId, clusterName, namespaceName, branchName, maxId.get());
+              appId, clusterName, namespaceName, branchName, maxId.get());
+
       Set<Long> releaseIds = cleanReleaseHistoryList.stream()
-          .map(ReleaseHistory::getReleaseId)
-          .collect(Collectors.toSet());
+              .map(ReleaseHistory::getReleaseId)
+              .collect(Collectors.toSet());
```

Comment:
This change appears unnecessary.
```suggestion
          .map(ReleaseHistory::getReleaseId)
          .collect(Collectors.toSet());
```